scalafix-cli/src/main/scala/scalafix/internal/cli/ClasspathOps.scala

```diff
@@ -22,11 +22,15 @@ object ClasspathOps {
   /** Process classpath with metacp to build semanticdbs of global symbols. **/
   def toMetaClasspath(
       sclasspath: Classpath,
-      cacheDirectory: Option[AbsolutePath],
-      parallel: Boolean, // unused until we upgrade scalameta for https://github.com/scalameta/scalameta/pull/1474
-      out: PrintStream): Option[Classpath] = {
+      cacheDirectory: Option[AbsolutePath] = None,
+      parallel: Boolean = true, // unused until we upgrade scalameta for https://github.com/scalameta/scalameta/pull/1474
```

Comment:
Still unused?

Reply:
Was unused, but I just updated to make use of it.